### PR TITLE
feat: additional parameter support

### PR DESCRIPTION
**Pull request by Aayush-Abhyarthi**
### Description

Cloudant module missed some parameters as argument reference. This PR adds those.

https://github.ibm.com/GoldenEye/issues/issues/4584

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Comment by Aayush-Abhyarthi** [comment]
The upgrade test failed with the following error:-
Screenshot 2024-02-20 at 3 09 34 AM
resources were getting updated

File: variables.tf [hunk 1]
```diff
@@ -25,6 +25,24 @@ variable "instance_name" {
...
+variable "origins" {
+  description = "An array of strings that contain allowed origin domains. "
+  type        = list(string)
+  default     = []
+}
+
+variable "enable_cors" {
+  description = "Boolean value to enable CORS. The supported values are true and false."
+  type        = bool
+  default     = true
```
Comment (ocofaigh) [inline variables.tf:43]: I would default this `false` and create a commit with `UNSKIP UPGRADE TEST` so we dont have to skip it

**Review by ocofaigh — CHANGES_REQUESTED** [review]
These changed need to be also made in `modules/fscloud` and `solutions/dedicated`

**Comment by Aayush-Abhyarthi** [comment]
Screenshot 2024-02-29 at 3 07 49 PM

even after making the default value of the enable_cors as false, the infrastructure is getting updated

**Comment by ocofaigh** [comment]
@Aayush-Abhyarthi thanks for checking - since its an update in place, its fine to skip the upgrade test as long as we include it in the release notes. 
Since you unskipped the upgrade test, you may not be able to skip it again. If this is the case can you create a new PR please?

**Comment by Aayush-Abhyarthi** [comment]
https://github.com/terraform-ibm-modules/terraform-ibm-cloudant/pull/121

**Pull request closed without merging**